Web/Case Study Assignment-3/CarDekho/home.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>CarDekho</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" type="text/css" href="style.css">
+<link rel="stylesheet" type="text/css" href="styleLogin.css">
+
+<script>
+	window.history.forward();
+	function query() {
+		var i = document.URL.indexOf('?');
+		var p = [];
+		if (i != -1) {
+			var p1 = document.URL.substring(i + 1, document.URL.length).split(
+					'&');
+			var nm = [];
+			for ( var i = 0; i < p1.length; i++) {
+				nm = p1[i].split('=');
+				p[nm[0]] = nm[1];
+			}
+
+		}
+		return p;
+	}
+</script>
+</head>
+<body>
+	<header>
+		<br />
+		<span id="logo">Car Dekho</span>
+		<div id="user">
```

Comment:
ACTION: If user is not logged in then it will show "undefined". Please check it.